subsys/bluetooth/audio/pacs.c

```diff
@@ -257,6 +259,20 @@ static int set_available_contexts(uint16_t contexts, uint16_t *available,
 	return 0;
 }
 
+static int set_supported_contexts(uint16_t contexts, uint16_t *supported)
+{
+	int err;
+
+	*supported = contexts;
+
+	err = k_work_schedule(&supported_contexts_work, PAC_NOTIFY_TIMEOUT);
+	if (err < 0) {
+		return err;
+	}
+
+	return 0;
```

Comment:
I've followed the pattern from available contexts handling, what do you propose? Should we return here without giving an error?

Reply:
That is a good question. 

I am thinking that we should either 
A) Only set the `supported` value if `k_work_reschedule` returns 0 or above
B) Only log it as a warning if `k_work_reschedule` returns `< 0` and do not return an error (I guess the function becomes `void` then)
c) Keep as is (as that is how available contexts is implemented as you said). 

@MariuszSkamra any input?

Reply:
That makes sense to me, but it also (unfortunately) not how much of our implementations otherwise work (if notification is failed to be sent for whatever reason, we typically just continue) - But maybe that needs to change?

Reply:
I think this might be a topic for a discussion for a weekly meeting, but anyway I've changed it now so supported contexts are changed after successful notification `k_work_reschedule`.